lib/community/widgets/post_card_view_comfortable.dart

```diff
@@ -221,7 +220,7 @@ class PostCardViewComfortable extends StatelessWidget {
                 textContent,
                 maxLines: 4,
                 overflow: TextOverflow.ellipsis,
-                textScaleFactor: MediaQuery.of(context).textScaleFactor * state.contentFontSizeScale.textScaleFactor,
+                textScaler: TextScaler.linear(state.contentFontSizeScale.textScaleFactor),
```

Comment:
Any reason for not using `ScalableText` here?

Reply:
Oops, I forgot to change this instance!